package.json

```diff
@@ -66,6 +66,18 @@
     "jest-diff": "^29.7.0",
     "lodash": "^4.17.21"
   },
+  "peerDependencies": {
+    "sass": ">=1.45.0",
+    "sass-embedded": ">=1.45.0"
```

Comment:
I'm curious why `">=1.45.0"`. I think it's the minimum supported version? Should we add that to the README?

Reply:
Yeah, I think that's the minimum version that supports the "modern" API (which is what True uses). Added a note in https://github.com/oddbird/true/commit/51c9e419ee5c2c261039dbe740d673fb9687cbad